data/json/npcs/talk_tags.json

```diff
@@ -967,7 +1252,21 @@
       "Is this Reno?  Because I need to watch you die!",
       "You're going to pay for that, <name_b>!",
       "You're gonna rot in hell, you pieces of shit!",
-      "You're gonna rot in hell for this!"
+      "You're gonna rot in hell for this!",
+      "You're going to die here, you <swear> <name_b>",
+      "Your death will make me <happy>, you <swear!>",
+      "You ain't the first, and probably not the last, you <swear!>",
+      "Better you than me, you <swear> <swear!>",
+      "I hope your shit sells for a good price, you <name_b>!",
+      "I'll go easy on you if you quit running like a bitch!",
+      "Come at me, you little <swear!>",
+      "I've fought small blobs fiercer than you!",
+      "I don't mind getting blood on my boots, you <swear!>",
+      "You're nothing to me but just another target!",
+      "Come on, fight back, make me laugh!",
+      "Ain't gonna forget my first kill.",
+      "Quit running, I hate moving targets!",
+      "Fuck you, you <swear!>"
```

Comment:
`<swear!>` has things like `damn`, `damnit`, `bloody hell`.  I think you need to switch to `<name_b>` for these contexts.

Reply:
Yup, you're right, I'll change this one.